src/pages/journal/components/test-slot/test-slot.html

```diff
@@ -1,21 +1,54 @@
-<ion-card>
-  <ion-row align-items-center>
-    <ion-col col-auto>
-      <indicators [showSpecialNeedsIndicator]="isSpecialNeedsSlot()"></indicators>
+<ion-card class="test-slot-card" [ngClass]="{'test-slot-portrait-mode': isPortrait()}">
+  <ion-row align-items-center class="test-slot-row" nowrap>
+    <ion-col col-auto padding-left>
+      <indicators
+        [showSpecialNeedsIndicator]="isSpecialNeedsSlot()"
+      >
+      </indicators>
     </ion-col>
     <ion-col col-2>
-      <time [time]="slot.slotDetail.start" [testComplete]="testComplete"></time>
+      <time
+        [time]="slot.slotDetail.start"
+        [testComplete]="testComplete"
+      >
+    </time>
     </ion-col>
-    <ion-col col-5>
+    <ion-col col-4 style="margin-left: -30px;">
```

Comment:
Would be nice to get this style out of the template.

Reply:
done